crates/turborepo-repository/src/package_graph/builder.rs

```diff
@@ -15,18 +15,20 @@ use turborepo_lockfiles::Lockfile;
 
 use super::{PackageGraph, WorkspaceInfo, WorkspaceName, WorkspaceNode};
 use crate::{
+    discovery::{LocalPackageDiscovery, PackageDiscovery},
     package_graph::{PackageName, PackageVersion},
     package_json::PackageJson,
     package_manager::PackageManager,
 };
 
-pub struct PackageGraphBuilder<'a> {
+pub struct PackageGraphBuilder<'a, T> {
     repo_root: &'a AbsoluteSystemPath,
     root_package_json: PackageJson,
     is_single_package: bool,
-    package_manager: Option<PackageManager>,
+    package_manager: PackageManager,
```

Comment:
Not blocking, but now that this information is returned from `PackageDiscovery::discover_packages` it feels like `package_manager` should come from there instead of being passed in.